Scarb.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cairo_wadray"
+version = "0.1.0"
+
+[dependencies]
+starknet = "2.3.1"
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.11.0" }
```

Comment:
Foundry should be a [`[dev-dependencies]`](https://docs.swmansion.com/scarb/docs/reference/specifying-dependencies.html#development-dependencies)